server/test/follow.spec.js

```diff
@@ -0,0 +1,149 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+
+chai.use(chaiHttp);
+
+let userToken;
+
+describe('Follow/Unfollow Test Feature', () => {
+  describe('When User login', () => {
```

Comment:
This extra `describe block not necessary`

Reply:
i have done this.